--- a/cmake/modules/FindXBase.cmake
+++ b/cmake/modules/FindXBase.cmake
@@ -12,13 +12,13 @@
 	SET(XBASE_FOUND TRUE)
 
 else ()
 
-       find_path(XBase_INCLUDE_DIR NAMES xbase.h
-        PATHS /usr/include/xbase /usr/local/include/xbase
+       find_path(XBase_INCLUDE_DIR NAMES xbase64.h
+        PATHS /usr/include/xbase64 /usr/local/include/xbase
        )
 
-       find_library(XBase_LIBRARIES NAMES xbase )
+       find_library(XBase_LIBRARIES NAMES xbase64 )
 
        include(FindPackageHandleStandardArgs)
        find_package_handle_standard_args(XBase DEFAULT_MSG XBase_INCLUDE_DIR XBase_LIBRARIES )
 
--- a/kexi/kexidb/drivers/xbase/xbaseexport.cpp
+++ b/kexi/kexidb/drivers/xbase/xbaseexport.cpp
@@ -32,9 +32,9 @@
 #include <migration/keximigratedata.h>
 
 #include <cstring>
 
-#include "xbase.h"
+#include "xbase64.h"
 
 using namespace KexiDB;
 
 class KexiDB::xBaseExportPrivate {
--- a/kexi/migration/xbase/xbasemigrate.h
+++ b/kexi/migration/xbase/xbasemigrate.h
@@ -22,9 +22,9 @@
 
 #include <QHash>
 
 #include <migration/keximigrate.h>
-#include "xbase.h"
+#include "xbase64.h"
 
 namespace KexiMigration
 {
 
--- a/kexi/migration/xbase/xbasemigrate.cpp
+++ b/kexi/migration/xbase/xbasemigrate.cpp
@@ -253,18 +253,18 @@
           blobFieldLength = tableDbf->GetMemoFieldLen(j);
           memoBuffer = new char[blobFieldLength];
 
           #ifdef XB_LOCKING_ON
-            tableDbf->LockMemoFile( F_SETLK, F_RDLCK );
+            // tableDbf->LockMemoFile( F_SETLK, F_RDLCK );
           #endif
 
           if ( ( returnCode = tableDbf->GetMemoField( j , blobFieldLength, memoBuffer, F_SETLKW ) ) != XB_NO_ERROR ) {
             kDebug()<<"Error reading blob field. Error code: "<<returnCode; // make error message more verbose
           } else {
             val = KexiDB::cstringToVariant( memoBuffer, fieldsExpanded.at(j)->field, blobFieldLength );
           }
           #ifdef XB_LOCKING_ON
-            tableDbf->LockMemoFile( F_SETLK, F_UNLCK );
+            // tableDbf->LockMemoFile( F_SETLK, F_UNLCK );
           #endif
 
           break;
         #else
