Index: calligra/plugins/colorengines/lcms2/colorprofiles/data/CMakeLists.txt
===================================================================
--- calligra.orig/plugins/colorengines/lcms2/colorprofiles/data/CMakeLists.txt	2014-07-15 15:01:36.007724217 +0200
+++ calligra/plugins/colorengines/lcms2/colorprofiles/data/CMakeLists.txt	2014-07-15 15:01:36.003724384 +0200
@@ -1,4 +0,0 @@
-install(FILES
-    CMY.icm 
-    fogra27l.icm
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/color/icc/pigment)
Index: calligra/plugins/colorengines/lcms2/CMakeLists.txt
===================================================================
--- calligra.orig/plugins/colorengines/lcms2/CMakeLists.txt	2014-07-15 15:01:32.683863328 +0200
+++ calligra/plugins/colorengines/lcms2/CMakeLists.txt	2014-07-15 15:02:55.052417218 +0200
@@ -5,8 +5,6 @@
 	add_subdirectory(tests)
 endif(!WIN32)
 
-add_subdirectory(colorprofiles)
-
 include_directories( ${PIGMENT_INCLUDES} ${LCMS2_INCLUDE_DIR} )
 
 include_directories( 
