--- a/krita/image/filter/kis_filter_registry.cc
+++ b/krita/image/filter/kis_filter_registry.cc
@@ -50,9 +50,9 @@
 }
 
 KisFilterRegistry* KisFilterRegistry::instance()
 {
-    KisFilterRegistry *reg = qApp->findChild<KisFilterRegistry *>("");
+    static KisFilterRegistry *reg = qApp->findChild<KisFilterRegistry *>("");
     if (!reg) {
         reg = new KisFilterRegistry(qApp);
         KoPluginLoader::instance()->load("Krita/Filter", "Type == 'Service' and ([X-Krita-Version] == 28)");
     }
