--- a/krita/gemini/MainWindow.cpp
+++ b/krita/gemini/MainWindow.cpp
@@ -166,8 +166,13 @@
         sketchView->engine()->addImportPath(appdir.canonicalPath() + "/lib/calligra/imports");
         sketchView->engine()->addImportPath(appdir.canonicalPath() + "/lib64/calligra/imports");
         QString mainqml = appdir.canonicalPath() + "/share/apps/kritagemini/kritagemini.qml";
 #else
+
+#ifdef LIBRARY_ARCHITECTURE
+        sketchView->engine()->addImportPath(KGlobal::dirs()->findDirs("lib", LIBRARY_ARCHITECTURE "/calligra/imports").value(0));
+#endif
+
         sketchView->engine()->addImportPath(KGlobal::dirs()->findDirs("lib", "calligra/imports").value(0));
         QString mainqml = KGlobal::dirs()->findResource("data", "kritagemini/kritagemini.qml");
 #endif
 
--- a/krita/sketch/CMakeLists.txt
+++ b/krita/sketch/CMakeLists.txt
@@ -142,8 +142,11 @@
 # =============================================================================
 
 kde4_add_executable(kritasketch ${kritasketch_SRCS})
 
+set_target_properties(kritasketch PROPERTIES COMPILE_DEFINITIONS
+    LIBRARY_ARCHITECTURE="${CMAKE_LIBRARY_ARCHITECTURE}")
+
 target_link_libraries(kritasketch
     ${QT_QTDECLARATIVE_LIBRARY}
     ${QT_QTGUI_LIBRARIES}
     ${QT_QTCORE_LIBRARIES}
--- a/krita/sketch/MainWindow.cpp
+++ b/krita/sketch/MainWindow.cpp
@@ -121,8 +121,13 @@
     view->engine()->addImportPath(appdir.canonicalPath() + "/lib/calligra/imports");
     view->engine()->addImportPath(appdir.canonicalPath() + "/lib64/calligra/imports");
     QString mainqml = appdir.canonicalPath() + "/share/apps/kritasketch/kritasketch.qml";
 #else
+
+#ifdef LIBRARY_ARCHITECTURE
+    view->engine()->addImportPath(KGlobal::dirs()->findDirs("lib", LIBRARY_ARCHITECTURE "/calligra/imports").value(0));
+#endif
+
     view->engine()->addImportPath(KGlobal::dirs()->findDirs("lib", "calligra/imports").value(0));
     QString mainqml = KGlobal::dirs()->findResource("data", "kritasketch/kritasketch.qml");
 #endif
 
--- a/krita/gemini/CMakeLists.txt
+++ b/krita/gemini/CMakeLists.txt
@@ -30,8 +30,11 @@
 # =============================================================================
 
 kde4_add_executable(kritagemini ${kritagemini_SRCS})
 
+set_target_properties(kritagemini PROPERTIES COMPILE_DEFINITIONS
+    LIBRARY_ARCHITECTURE="${CMAKE_LIBRARY_ARCHITECTURE}")
+
 target_link_libraries(kritagemini
     ${QT_QTDECLARATIVE_LIBRARY}
     ${QT_QTGUI_LIBRARIES}
     ${QT_QTCORE_LIBRARIES}
