Index: calligra-2.7.5/kexi/CMakeLists.txt
===================================================================
--- calligra-2.7.5.orig/kexi/CMakeLists.txt	2013-11-24 14:49:21.000000000 +0100
+++ calligra-2.7.5/kexi/CMakeLists.txt	2013-12-06 21:31:21.000000000 +0100
@@ -94,7 +94,7 @@
 INCLUDE (FindPostgreSQL)
 
 set (PQXX_MIN_VERSION "3.0.0")
-set (PQXX_MAX_VERSION "4.0.0")
+set (PQXX_MAX_VERSION "5.0.0")
 
 macro_optional_find_package(PostgreSQL)
 macro_log_feature(POSTGRESQL_FOUND "libpq" "C application programmer's interface to PostgreSQL" "http://www.postgresql.org" FALSE "" "Required by Kexi PostgreSQL driver")
