Index: calligra-2.9.5/CMakeLists.txt
===================================================================
--- calligra-2.9.5.orig/CMakeLists.txt
+++ calligra-2.9.5/CMakeLists.txt
@@ -73,7 +73,7 @@ endif()
 if(NOT DEFINED RELEASE_BUILD)
     # estimate mode by CMAKE_BUILD_TYPE content if not set on cmdline
     string(TOLOWER "${CMAKE_BUILD_TYPE}" CMAKE_BUILD_TYPE_TOLOWER)
-    set(RELEASE_BUILD_TYPES "release" "relwithdebinfo" "minsizerel")
+    set(RELEASE_BUILD_TYPES "release" "relwithdebinfo" "minsizerel" "debian")
     list(FIND RELEASE_BUILD_TYPES "${CMAKE_BUILD_TYPE_TOLOWER}" INDEX)
     if (INDEX EQUAL -1)
         set(RELEASE_BUILD FALSE)
