Index: calligra-2.9.5/krita/sketch/draganddrop/CMakeLists.txt
===================================================================
--- calligra-2.9.5.orig/krita/sketch/draganddrop/CMakeLists.txt
+++ calligra-2.9.5/krita/sketch/draganddrop/CMakeLists.txt
@@ -20,6 +20,6 @@ qt4_automoc(${declarativedragdrop_SRCS})
 kde4_add_library(draganddropplugin SHARED ${declarativedragdrop_SRCS})
 target_link_libraries(draganddropplugin ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTDECLARATIVE_LIBRARY})
 
-install(TARGETS draganddropplugin DESTINATION ${CMAKE_INSTALL_LIBDIR}/calligra/imports/org/krita/draganddrop)
+install(TARGETS draganddropplugin DESTINATION /usr/lib/calligra/imports/org/krita/draganddrop)
 
-install(FILES qmldir DESTINATION ${CMAKE_INSTALL_LIBDIR}/calligra/imports/org/krita/draganddrop)
+install(FILES qmldir DESTINATION /usr/lib/calligra/imports/org/krita/draganddrop)
