Index: calligra-2.7.5/kexi/CMakeLists.txt
===================================================================
--- calligra-2.7.5.orig/kexi/CMakeLists.txt	2013-11-27 19:24:35.504323078 +0000
+++ calligra-2.7.5/kexi/CMakeLists.txt	2013-11-27 19:24:35.504323078 +0000
@@ -94,7 +94,6 @@
 INCLUDE (FindPostgreSQL)
 
 set (PQXX_MIN_VERSION "3.0.0")
-set (PQXX_MAX_VERSION "4.0.0")
 
 macro_optional_find_package(PostgreSQL)
 macro_log_feature(POSTGRESQL_FOUND "libpq" "C application programmer's interface to PostgreSQL" "http://www.postgresql.org" FALSE "" "Required by Kexi PostgreSQL driver")
@@ -102,11 +101,11 @@
 pkg_check_modules (PQXX libpqxx)
 
 IF (POSTGRESQL_FOUND AND PQXX_FOUND)
-   IF (PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION AND PQXX_VERSION VERSION_LESS PQXX_MAX_VERSION)
+   IF (PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION)
       macro_log_feature(PQXX_FOUND "libpqxx" "Official C++ client API for PostgreSQL" "http://pqxx.org/development/libpqxx/" FALSE "" "Required by Kexi PostgreSQL driver")
-   ELSE (PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION AND PQXX_VERSION VERSION_LESS PQXX_MAX_VERSION)
-      macro_log_feature(FALSE "libpqxx" "Official C++ client API for PostgreSQL, version >= ${PQXX_MIN_VERSION} and older than ${PQXX_MAX_VERSION}" "http://pqxx.org/development/libpqxx/" FALSE "" "Required by Kexi PostgreSQL driver")
-   ENDIF (PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION AND PQXX_VERSION VERSION_LESS PQXX_MAX_VERSION)
+   ELSE (PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION)
+      macro_log_feature(FALSE "libpqxx" "Official C++ client API for PostgreSQL, version >= ${PQXX_MIN_VERSION}" "http://pqxx.org/development/libpqxx/" FALSE "" "Required by Kexi PostgreSQL driver")
+   ENDIF (PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION)
 ENDIF (POSTGRESQL_FOUND AND PQXX_FOUND)
 
 macro_optional_find_package(FreeTDS)
Index: calligra-2.7.5/kexi/kexidb/drivers/CMakeLists.txt
===================================================================
--- calligra-2.7.5.orig/kexi/kexidb/drivers/CMakeLists.txt	2013-11-24 13:09:38.000000000 +0000
+++ calligra-2.7.5/kexi/kexidb/drivers/CMakeLists.txt	2013-11-27 21:09:31.927938334 +0000
@@ -7,9 +7,9 @@
    add_subdirectory(mysql)
 ENDIF (MYSQL_FOUND)
 
-IF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION AND PQXX_VERSION VERSION_LESS PQXX_MAX_VERSION)
+IF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION)
    add_subdirectory(pqxx)
-ENDIF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION AND PQXX_VERSION VERSION_LESS PQXX_MAX_VERSION)
+ENDIF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION)
 
 IF (FREETDS_FOUND)
    add_subdirectory(sybase)
Index: calligra-2.7.5/kexi/migration/CMakeLists.txt
===================================================================
--- calligra-2.7.5.orig/kexi/migration/CMakeLists.txt	2013-11-24 13:09:38.000000000 +0000
+++ calligra-2.7.5/kexi/migration/CMakeLists.txt	2013-11-28 06:07:01.472584445 +0000
@@ -12,9 +12,9 @@
   add_subdirectory( mysql )
 ENDIF (MYSQL_FOUND)
 
-IF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION AND PQXX_VERSION VERSION_LESS PQXX_MAX_VERSION)
+IF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION)
   add_subdirectory( pqxx )
-ENDIF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION AND PQXX_VERSION VERSION_LESS PQXX_MAX_VERSION)
+ENDIF (POSTGRESQL_FOUND AND PQXX_FOUND AND PQXX_VERSION VERSION_GREATER PQXX_MIN_VERSION)
 
 IF(FREETDS_FOUND)
   add_subdirectory(sybase)
