commit d0a0a91947dbd40ae13af0f960ef0705825c0a5c
Author: Rex Dieter <rdieter@math.unl.edu>
Date:   Fri Sep 4 07:45:53 2015 -0500

    fix arm FTBFS (and qreal != double assumptions)

diff --git a/krita/plugins/paintops/chalk/kis_chalk_paintop_settings.cpp b/krita/plugins/paintops/chalk/kis_chalk_paintop_settings.cpp
index 5b1eddd..f6cfd67 100644
--- a/krita/plugins/paintops/chalk/kis_chalk_paintop_settings.cpp
+++ b/krita/plugins/paintops/chalk/kis_chalk_paintop_settings.cpp
@@ -51,7 +51,7 @@ QPainterPath KisChalkPaintOpSettings::brushOutline(const KisPaintInformation &in
 
         QPainterPath tiltLine;
         QLineF tiltAngle(QPointF(0.0,0.0), QPointF(0.0,size));
-        tiltAngle.setLength(qMax(size*0.5, qreal(50.0)) * (1 - info.tiltElevation(info, 60.0, 60.0, true)));
+        tiltAngle.setLength(qMax(size*qreal(0.5), qreal(50.0)) * (1 - info.tiltElevation(info, 60.0, 60.0, true)));
         tiltAngle.setAngle((360.0 - fmod(KisPaintInformation::tiltDirection(info, true) * 360.0 + 270.0, 360.0))-3.0);
         tiltLine.moveTo(tiltAngle.p1());
         tiltLine.lineTo(tiltAngle.p2());
diff --git a/krita/plugins/paintops/libpaintop/kis_brush_based_paintop_settings.cpp b/krita/plugins/paintops/libpaintop/kis_brush_based_paintop_settings.cpp
index fe1f24d..1798805 100644
--- a/krita/plugins/paintops/libpaintop/kis_brush_based_paintop_settings.cpp
+++ b/krita/plugins/paintops/libpaintop/kis_brush_based_paintop_settings.cpp
@@ -73,7 +73,7 @@ QPainterPath KisBrushBasedPaintOpSettings::brushOutlineImpl(const KisPaintInform
         QPainterPath realOutline = brush->outline();
         QPainterPath tiltLine;
         QLineF tiltAngle(realOutline.boundingRect().center(), realOutline.boundingRect().topLeft());
-        tiltAngle.setLength(qMax(realOutline.boundingRect().width()*0.5, qreal(50.0)) * (1 - info.tiltElevation(info, 60.0, 60.0, true)));
+        tiltAngle.setLength(qMax(realOutline.boundingRect().width()*qreal(0.5), qreal(50.0)) * (1 - info.tiltElevation(info, 60.0, 60.0, true)));
         tiltAngle.setAngle((360.0 - fmod(KisPaintInformation::tiltDirection(info, true) * 360.0 + 270.0, 360.0))-3.0);
         tiltLine.moveTo(tiltAngle.p1());
         tiltLine.lineTo(tiltAngle.p2());
diff --git a/krita/plugins/paintops/sketch/kis_sketch_paintop_settings.cpp b/krita/plugins/paintops/sketch/kis_sketch_paintop_settings.cpp
index 4af0b39..8136c60 100644
--- a/krita/plugins/paintops/sketch/kis_sketch_paintop_settings.cpp
+++ b/krita/plugins/paintops/sketch/kis_sketch_paintop_settings.cpp
@@ -62,7 +62,7 @@ QPainterPath KisSketchPaintOpSettings::brushOutline(const KisPaintInformation &i
         path = ellipseOutline(diameter, diameter, 1.0, 0.0/*brush->scale(), brush->angle()*/);
         QPainterPath tiltLine;
         QLineF tiltAngle(path.boundingRect().center(), path.boundingRect().topLeft());
-        tiltAngle.setLength(qMax(diameter*0.5, qreal(50.0)) * (1 - info.tiltElevation(info, 60.0, 60.0, true)));
+        tiltAngle.setLength(qMax(diameter*qreal(0.5), qreal(50.0)) * (1 - info.tiltElevation(info, 60.0, 60.0, true)));
         tiltAngle.setAngle((360.0 - fmod(KisPaintInformation::tiltDirection(info, true) * 360.0 + 270.0, 360.0))-3.0);
         tiltLine.moveTo(tiltAngle.p1());
         tiltLine.lineTo(tiltAngle.p2());
