From a56abd9bb8d370c9f28d15a09b6c435fef04fdd6 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Thu, 15 Sep 2011 22:41:54 +0200
Subject: [PATCH] GENERIC_KOFFICE_LIB_{,SO}VERSION -> GENERIC_CALLIGRA_LIB_{,SO}VERSION

---
 kexi/widget/dataviewcommon/CMakeLists.txt |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kexi/widget/dataviewcommon/CMakeLists.txt b/kexi/widget/dataviewcommon/CMakeLists.txt
index 262299e..1376aa2 100644
--- a/kexi/widget/dataviewcommon/CMakeLists.txt
+++ b/kexi/widget/dataviewcommon/CMakeLists.txt
@@ -29,5 +29,5 @@ target_link_libraries(
     ${QT_QT3SUPPORT_LIBRARY}
 )
 
-set_target_properties(kexidataviewcommon PROPERTIES VERSION ${GENERIC_KOFFICE_LIB_VERSION} SOVERSION ${GENERIC_KOFFICE_LIB_SOVERSION} )
+set_target_properties(kexidataviewcommon PROPERTIES VERSION ${GENERIC_CALLIGRA_LIB_VERSION} SOVERSION ${GENERIC_CALLIGRA_LIB_SOVERSION} )
 install(TARGETS kexidataviewcommon  ${INSTALL_TARGETS_DEFAULT_ARGS})
-- 
1.7.1

